1.9/api/logs.yaml

```diff
@@ -0,0 +1,216 @@
+swagger: '2.0'
+
+info:
+  version: "0.0.1"
+  title: DC/OS Log API
+  description: >
+    The first version of the DC/OS Log API.
+    Note that the endpoints in this spec are all *service-specific*. dcos-log
```

Comment:
This file is managed by the day2ops team, copied from the source repo: https://github.com/dcos/dcos-log/blob/master/swagger.yml